### DISCUSSION TITLE
Which version of Argo can be run on Kubernetes version 1.20?

### DISCUSSION THREAD

**Discussion by sunwuyan** [argoproj/argo-workflows#13758] [Q&A] [answered]
HI，the version of my kubernetes production cluster is 1.20. I want to install argo-workflows in it, but I don’t know which version is suitable? Can anyone tell me? Tks!

**Comment by jswxstw** [comment, marked as answer]
The current stable version, currently v3.5.11, works fine.
However, I suggest you upgrade your Kubernetes version because v1.20 has some bugs that will affect some functions of Argo.

**Reply by sunwuyan** [reply]
Great, 3q!

**Reply by agilgur5** [reply]
> I suggest you upgrade your Kubernetes version because v1.20 has some bugs that will affect some functions of Argo.

And because it is EoL (for almost 4 years) in general and does not get security patches etc.
